make_docs.nu

```diff
@@ -243,10 +242,13 @@ $"## Notes
         # FIXME: Parentheses are required here to mutate $input_output, otherwise it won't work, maybe a bug?
         $input_output = ($input_output | append [[input output]; [$input $output]])
     }
-    let in_out = if ($input_output | length) > 0 {
-        let markdown = ($input_output | sort-by input | to md --pretty | str replace -a '<' '\<' | str replace -a '>' '\>')
-        ['', '## Input/output types:', '', $markdown, ''] | str join (char newline)
-    } else { '' }
+    # Input/output types: use help commands
+    let input_output_table = help commands | where name == $command.name | get input_output | first | to md
```

Comment:
```suggestion
    let input_output_table = (
        help commands
        | where name == $command.name
        | get input_output
        | first
        | to md --pretty
        | str replace -a '<' '&lt;'
        | str replace -a '>' '&gt;'
    )
```

This replicates the previous behavior (old line 247), and also switches the `<`/`>` to their HTML entity counterparts instead of just escaping them with `\`